### PR TITLE
Fix titles to allow users to show them on GitHub

### PR DESCRIPTION
**Pull request by akito19**
Use `hide_title` to enable h1 as `#` with markdown.
It will be trivial changes for users who see our docs on help.sider.review, but it will help for users who do it on GitHub.com.

See: https://github.com/sider/sider-docs/pull/98#discussion_r254556951

### CODE REVIEW

File: website/static/css/custom.css [hunk 1]
```diff
@@ -21,7 +21,7 @@ a:hover {
 }
 
 /* To fix h1 position */
-header.postHeader:empty + article h1 {
+article h1 {
```
Comment (akito19) [inline website/static/css/custom.css]: **before**

screen shot 2019-02-12 at 14 56 28

**after**

screen shot 2019-02-12 at 14 56 40

**Comment by ybiquitous** [comment]
> If this field is not present, the document's `sidebar_label` will default to its `title`.

What about removing `sidebar_label` same as `title`? I think we can reduce bugs that `title` and `sidebar_label` is different.

**Comment by akito19** [comment]
I'd like to set `sidebar_label` explicitly even if `title` and `sidebar_label` is the same.
This is because we have had a case that these values are different, https://github.com/sider/sider-docs/blob/master/docs/enterprise/testing/guide.md, and I believe it is worth showing what `sidebar_label` key exists for any users.

**Comment by ybiquitous** [comment]
OK, I see. 🙆‍♂️

File: docs/advanced-settings/inline-comments.md [hunk 2]
```diff
@@ -16,6 +19,4 @@ If an analysis tool finds more than 5 issues, a summary of the issues will be po
 
 ![Inline Comments Settings](../assets/inline-comments-setting.png)
 
-{% hint style="warning" %}
-If the Test Mode section is set to `ON`, Sider will post comments to your pull request. Note that this interface requires Sider to be enable as a GitHub App.
-{% endhint %}
+> If the Test Mode section is set to `ON`, Sider will post comments to your pull request. Note that this interface requires Sider to be enable as a GitHub App.
```
Comment (ybiquitous) [inline docs/advanced-settings/inline-comments.md]: `<blockquote>` has a different semantics with normal `<div>`. But markdown doesn't have a correct syntax in this case... 🤔

File: docs/advanced-settings/private-dependencies.md [hunk 3]
```diff
@@ -2,8 +2,11 @@
...
 title: Private Dependencies
 sidebar_label: Private Dependencies
+hide_title: true
 ---
 
+# Private Dependencies
+
 Analyzing a private project sometimes needs access to another private repository. Your team might be using a Git repository to distribute a private library. This kind of dependency is supported in some tools including Bundler, npm.
 
 ## Private Dependencies
```
Comment (ybiquitous) [inline docs/advanced-settings/private-dependencies.md]: [IMO] Duplicate headings produce broken link such as <https://help.sider.review/foo/bar#private-dependencies>.

To detect such problems, I opened the PR #108.

File: docs/introduction.md [hunk 4]
```diff
@@ -2,8 +2,11 @@
...
 title: Introduction
 sidebar_label: Introduction
+hide_title: true
 ---
 
+# Introduction
+
 ## About the documentation
 
 This is guides for Sider, analyzers and customization.
```
Comment (ybiquitous) [inline docs/introduction.md]: Is this a typo? `confortably` => `comfortably`

```diff
-We have prepared contents to use Sider confortably.
+We have prepared contents to use Sider comfortably.
```

Comment (akito19) [reply docs/introduction.md]: Yes.

Comment (akito19) [reply docs/introduction.md]: I found many typos with `misspell`...
I'll fix them by another pull request. I didn't understand why Sider hadn't indicated them when committing 😭

Comment (akito19) [reply docs/introduction.md]: (Almost are `node_modules`...)

**Pull request merged**